### ISSUE TITLE
Farbverlaub der Navigation & Menü

### ISSUE THREAD

**Issue by abergy** [FHBielefeld-IFM-WS1718-SWEng1/iOSApp#14]
Hier gibt es noch einen Fehler mit dem Anmelde-Button. Sobald ein Verlauf hinterlegt wird, ändert dieser die Breite

**Comment by abergy** [comment]
@dleunig ist das schon fertig ?

**Comment by dleunig** [comment]
Nein, das werde ich noch beheben, das ist ein Fehler der bei verschiedenen Gerätebreiten auftritt. Darum habe ich den erstmal auskommentiert.
